### PR TITLE
Fixed Alpha Blend for Path Search Debug Overlay

### PR DESCRIPTION
**Pull request by MutterOberin**
Fixed Path Cost Overlay Rendering.

Extends #8249

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/World/PathfinderDebugOverlay.cs [hunk 1]
```diff
@@ -72,14 +76,20 @@ public void Render(WorldRenderer wr)
 						continue;
 
 					var w = Math.Max(0, Math.Min(layer[cell], 128));
+
 					if (doDim)
-						layer[cell] = layer[cell] * 5 / 6;
+                        layer[cell] = layer[cell] * 5 / 6;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/World/PathfinderDebugOverlay.cs]: Please undo this change. We use tabs instead of spaces.

File: OpenRA.Mods.Common/Traits/World/PathfinderDebugOverlay.cs [hunk 2]
```diff
@@ -72,14 +76,20 @@ public void Render(WorldRenderer wr)
...
 
 					// TODO: This doesn't make sense for isometric terrain
 					var pos = wr.World.Map.CenterOfCell(cell);
 					var tl = wr.ScreenPxPosition(pos - new WVec(512, 512, 0));
 					var br = wr.ScreenPxPosition(pos + new WVec(511, 511, 0));
-					qr.FillRect(RectangleF.FromLTRB(tl.X, tl.Y, br.X, br.Y), Color.FromArgb(w, c));
+
+                    // Premultiplied Alpha Color Values:
+                    // Scaling Color Values by (w / 255)
+                    var c1 = Color.FromArgb((int)((w / 255f) * c.R), (int)((w / 255f) * c.G), (int)((w / 255f) * c.B));
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/World/PathfinderDebugOverlay.cs]: Also here: Please change this to tabs, we don't use spaces.

File: OpenRA.Mods.Common/Traits/World/PathfinderDebugOverlay.cs [hunk 3]
```diff
@@ -58,7 +60,9 @@ public void Render(WorldRenderer wr)
 
 			var qr = Game.Renderer.WorldQuadRenderer;
 			var doDim = refreshTick - world.WorldTick <= 0;
-			if (doDim) refreshTick = world.WorldTick + 20;
+
+			if (doDim)
+                refreshTick = world.WorldTick + 20;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/World/PathfinderDebugOverlay.cs]: And here.

File: mods/ts/rules/nod-support.yaml [hunk 4]
```diff
@@ -53,7 +53,7 @@ NALASR:
 	Turreted:
 		ROT: 10
 		InitialFacing: 300
-		Offset: 298,-171,0
+		Offset: 298,-171,300
```
Comment (abcdefg30) [inline mods/ts/rules/nod-support.yaml]: This has nothing to do with this PR?

**Comment by abcdefg30** [comment]
Please rebase this PR against bleed (I hope https://help.github.com/articles/about-git-rebase/ helps)
and get rid of those "Merged upstreamBleed" commits.
If you have problems with that, feel free to join our IRC channel and ask questions. ;)
(See http://www.openra.net/community/)

**Comment by MutterOberin** [comment]
Thanks for the comments!

First pull request ever, sorry if there are code style violations and unnecessary commit messages.

You can delete the whole pull request since #8248 solves the overlays in a smarter way.

**Comment by abcdefg30** [comment]
> You can delete the whole pull request since #8248 solves the overlays in a smarter way.

Hm, ok. I hope you get the chance to contribute in another pr though. :)
